src/screens/send/components/address-book/address-book.tsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import { View, Image, TouchableOpacity, SafeAreaView, SectionList, Alert } from 'react-native';
+import stylesProvider from './styles';
+import { IReduxState } from '../../../../redux/state';
+import { withTheme, IThemeProps } from '../../../../core/theme/with-theme';
+import { Icon } from '../../../../components/icon';
+import { connect } from 'react-redux';
+import { smartConnect } from '../../../../core/utils/smart-connect';
+import { Text } from '../../../../library';
+import { IContactsState, IContactState } from '../../../../redux/contacts/state';
+import { selectContacts } from '../../../../redux/contacts/selectors';
+import { formatAddress } from '../../../../core/utils/format-address';
+import { translate } from '../../../../core/i18n';
+import { Blockchain } from '../../../../core/blockchain/types';
+import Swipeable from 'react-native-gesture-handler/Swipeable';
+import { deleteContact, updateContactName } from '../../../../redux/contacts/actions';
+import { confirm } from '../../../../core/utils/dialog';
+import { IAccountState } from '../../../../redux/wallets/state';
+
+export interface IReduxProps {
+    contacts: IContactsState[];
+    deleteContact: typeof deleteContact;
+    updateContactName: typeof updateContactName;
+}
+
+export interface IExternalProps {
+    blockchain: Blockchain;
+    accounts: IAccountState[];
```

Comment:
you don't really need user accounts here, you only have to display the contacts, and when a contacts is selected return the address an maybe the name of contact